var/spack/repos/builtin/packages/mvapich/package.py

```diff
@@ -120,6 +120,8 @@ class Mvapich(AutotoolsPackage):
     with when("process_managers=auto"):
         conflicts("pmi_version=pmi2")
 
+    patch("slurm.patch")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
@murez, could you add the requested comment / provide the information about it please?